doc/sphinx/proofs/writing-proofs/reasoning-inductives.rst

```diff
@@ -982,7 +982,7 @@ This section describes some special purpose tactics to work with
 
    As :g:`H` occurs in the goal, we may want to reason by cases on its
    structure and so, we would like inversion tactics to substitute :g:`H` by
-   the corresponding @term in constructor form. Neither :tacn:`inversion` nor
+   the corresponding term in constructor form. Neither :tacn:`inversion` nor
```

Comment:
The intention may have been ```:n:`@term` ```, but I think the hyperlink is not needed here.  `term` is such as basic concept.